File: modul 7/soal 5/PRAK605-2210817220034-Wanda Karliyanti.c
#include <stdio.h>

int main(){
    int m1[10][10], m2[10][10], h[10][10];
    int i,j,k,n,jumlah=0;
    scanf("%d", &n);
    printf("Matriks A\n");
    for(i=0;i<n;i++){
        for(j=0;j<n;j++){
            scanf("%d", &m1[i][j]);
        }
    }
        printf("Matriks B\n");
        for(i=0;i<n;i++){
            for(j=0;j<n;j++){
                scanf("%d",&m2[i][j]);
            }
        }
            for(i=0;i<n;i++){
                for(j=0;j<n;j++){
                    for(k=0; k<n; k++){
                        jumlah=jumlah+m1[i][k]*m2[k][j];
                    }
                        h[i][j]= jumlah;
                        jumlah=0;
                }
            }
                        printf("\nMatriks AXB\n");
                        for(i=0;i<n;i++){
                            for(j=0;j<n;j++){
                                printf("%d ", h[i][j]);
                            }
                                printf("\n");
                        }
}

File: modul 7/soal 5/PRAK605-2210817220034-Wanda Karliyanti.py
n=int(input())
MA=[]
print("Matriks A")
for i in range (n):
    k=list(map(int, input().split()))
    MA.append(k)
MB=[]
print("Matriks B")
for i in range (n):
    k=list(map(int, input().split()))
    MB.append(k)
MAB=[]
print("\nMatriks AXB")
for i in range (n):
    MAB.append([])
    for j in range (n):
        x=0
        for k in range (n):
            x=x+MA[i][k]*MB[k][j]
        MAB[i].append(x)
for baris in MAB:
    for k in baris:
        print(k, end=' ')
    print()